app/validators/date_validator.rb

```diff
@@ -63,6 +66,15 @@ def future_date?(value)
     end
   end
 
+  def in_the_range?(value)
+    return false if within_range.nil?
+    if value.is_a?(String)
+      !((Date.parse(value) > 100.years.ago) and (Date.parse(value)) < 10.years.ago)
```

Comment:
The range should not be in the validator - it should be passed in to the validator.  

The only alternative is to rename the validator to be something like

DateTenToOneHundredYearsAndMinimumValidator which obviously you don't want to do - my point is, the validator is doing something that the name doesn't tell us it is doing ?